tests/atest/rules/misc/if-can-be-merged/invalid.robot

```diff
@@ -0,0 +1,16 @@
+*** Test Cases ***
+Missing condition
+    IF
+        Keyword
+    END
+    IF
+        Keyword
+    END
+
+Two conditions
+    IF    ${condition}    Keyword
+        Keyword
+    END
+    IF    ${condition}    Keyword
```

Comment:
Why this is not caught?

Reply:
It's invalid.robot file - containing invalid syntax. Currently (for 4.0) you cannot use inline syntax for IFs and when you do
```
IF    ${condition}    Keyword
```
RF thinks yp're trying to use two conditions (which is not allowed).